packages/wallet-backend-update/src/app.ts

```diff
@@ -71,7 +102,28 @@ export class App {
     const env = await this.container.resolve('env')
     const authController = await this.container.resolve('authController')
     const userController = await this.container.resolve('userController')
-
+    const paymentPointerController = await this.container.resolve(
+      'paymentPointerController'
+    )
+    const transactionController = await this.container.resolve(
+      'transactionController'
+    )
+    const incomingPaymentController = await this.container.resolve(
+      'incomingPaymentController'
+    )
+    const outgoingPaymentController = await this.container.resolve(
+      'outgoingPaymentController'
+    )
+    const rapydController = await this.container.resolve('rapydController')
+    const assetController = await this.container.resolve('assetController')
+    const accountController = await this.container.resolve('accountController')
+
+    app.use(
+      cors({
+        origin: 'http://localhost:4003',
```

Comment:
I think these settings were changes with the last Rico's PR, will be good to have them here as well

Reply:
gr8,thx